website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -61,25 +61,22 @@ This bug is fixed in Consul versions 1.13.1 and newer.
 
 #### Service mesh deployments using auto-encrypt or auto-config
 
-**Do not upgrade to Consul 1.13 yet** if using
-[auto-encrypt](/docs/agent/config/config-files#auto_encrypt) or
-[auto-config](/docs/agent/config/config-files#auto_config).
+Upgrade to **Consul version 1.13.2 or later**.
 
-In Consul 1.13, auto-encrypt and auto-config both cause Consul
+In Consul 1.13.1, auto-encrypt and auto-config both cause Consul
 to require TLS for gRPC communication with Envoy proxies.
 In environments where Envoy proxies are not already configured
-to use TLS for gRPC, upgrading Consul 1.13 will cause
+to use TLS for gRPC, upgrading Consul 1.13.1 will cause
 Envoy proxies to disconnect from the control plane (Consul agents).
 
-The underlying cause is the same as discussed in
-[deployments without the HTTPS port enabled on Consul agents](#service-mesh-deployments-without-the-https-port-enabled-on-consul-agents).
-However, when using auto-encrypt or auto-config,
-the problem **cannot** currently be avoided by 
-[modifying the agent's TLS configuration](#modify-the-consul-agent-s-tls-configuration)
-because auto-encrypt and auto-config automatically set
-interface-generic TLS configuration in a manner similar to
-[`tls.defaults`](/docs/agent/config/config-files#tls_defaults).
-We are working to address this problem in an upcoming 1.13 patch release.
+If upgrading to version 1.13.2 from 1.13.1 you must enable
```

Comment:
@freddygv :
- Does someone upgrading from <1.13 to 1.13.2+ need to touch `tls.grpc.use_auto_cert` at all? Or in some cases?
- Do we need to provide guidance on upgrading from 1.13.0/1 to 1.13.2? We had previously explicitly said "don't upgrade to 1.13 if using auto-config/encrypt", so I worry this paragraph is unnecessary detail that makes the guidance seem more complex.

Reply:
The motivation for adding this was a user incident where they upgraded from 1.13.1 to 1.13.2, and not using this new flag broke their setup. This set of circumstances is likely quite rare, but I erred on the side of documenting it because of someone having run into it.

Though this is actually relevant to someone upgrading from <= 1.13.1. The restriction about upgrading from 1.13.1 can be relaxed.